docs/HOL_Fault-Injection-for-Azure-IaaS-with-OMS/README.md

```diff
@@ -0,0 +1,322 @@
+
+1. Click on "Resource groups" icon, then on "+ Add".
+
+    ![](media/2.png)
+
+2. Enter a name for your resource group, pick a subscription, select a location and click on "Create".
+
+    ![](media/3.png)
+
+
+**Step 2.** Add "Log Analytics" to this resource group.
+
+1. Open the new resource group and click on "+ Add" button on the "Overview" page to add a new resource.
+
+    ![](media/4.png)  
+
+2. Filter resources by typing "Log Analytics" into the search bar, press "Enter", click "Log Analytics" and click on "Create" button.
+
+    ![](media/5.png)  
+
+3. Create an OMS workspace. Enter name, pick a subscription, location, current resource group and a pricing tier then click on "OK" button.
+
+    ![](media/6.png)  
+    >**Note:** OMS Workspace name has to be unique across Azure.
+
+**Step 3.** Get "Workspace ID" and "Primary Key" from OMS Portal.
+
+1. Navigate to your resource group and click on "Log Analytics" you just created.
+
+2. Click on "OMS Portal".
+
+    ![](media/7.png)  
+
+3. Once OMS portal has loaded, click on "Settings".
+
+    ![](media/8.png)  
+
+4. Click on "Connected Sources", then "Linux Servers" and note the "WORKSPACE ID" as well as the "PRIMARY KEY".
+
+    ![](media/1.png)
+
+**Step 4.** To set up collection of metrics click on "Data", "Linux Performance Counters", "Add the selected performance counters" and then on "Save".
+
+ ![](media/15.png)
+
+
+
+###Task 2: Deploy ARM template
+This task will setup the required environment in Azure.
+
+**Step 1.** Click on "Deploy to Azure" button below.
+
+[<img src="http://azuredeploy.net/deploybutton.png"/>](https://ms.portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FPartsUnlimitedMRP%2Fmaster%2FHOL_Fault-Injection-for-Azure-IaaS-with-OMS%2Fenv%2Ftemplate%2FlinuxVirtualMachineScaleSet.json)
+[<img src="http://armviz.io/visualizebutton.png"/>](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FPartsUnlimitedMRP%2Fmaster%2FHOL_Fault-Injection-for-Azure-IaaS-with-OMS%2Fenv%2Ftemplate%2FlinuxVirtualMachineScaleSet.json)
+
+
+**Step 2.** Select Azure Subscription and the resource group you created earlier. Fill in Settings section.
+
+  * "Vmss Name" should be a unique alpha-numeric name across Azure.
+  * Set "Instance Count" to 4.
+  * Admin username and passwords are the credentials for your virtual machines.
+  * Enter "Oms Workspace Id and the "Oms Primary Key" you took a note of in OMS Portal earlier.
+
+  ![](media/9.png)  
+
+**Step 3.** Reed and agree with terms and conditions, then click on "Purchase".
+
+  ![](media/10.png)  
+
+> **Note:** The deployment might take up to 10 minutes.
+
+**Step 4.** Once deployment has finished verify that MRP solution is being hosted and that all 4 machines connected to OMS.
+
+1. Navigate to your resource group and click on "Virtual Machine Scale Set" instance.
+
+    ![](media/11.png)  
+
```

Comment:
Please add a note here that the Type in the resource group will be shown as virtual machine scale sets.